src/helpers/helpers.canvas.js

```diff
@@ -135,7 +135,7 @@ export function drawPoint(ctx, options, x, y) {
       ctx.save();
       ctx.translate(x, y);
       ctx.rotate(rad);
-      ctx.drawImage(style, -style.width / 2, -style.height / 2, style.width, style.height);
+      ctx.drawImage(style, 0, -style.height / 2, style.width, style.height);
```

Comment:
What's the reason for this change? When this function is normally called, `x,y` is the center of the point and thus the center of the image

Reply:
Centering the image looks to be done because there is an assumed (fixed) width where the image will exist.

With this approach, because the space for the legend image is identical to the size of the legend image itself, there's no need to center it.

![Screenshot from 2021-08-11 13-39-22](https://user-images.githubusercontent.com/10779424/129030086-43363b53-3eb8-405b-9fb1-dbbfa60ed767.png)

That being said this was me hacking around and it resulted in no padding so centering might actually be a good idea if the "boxWidth" (legend icon width) is expected to include padding.